pandas/core/tools/datetimes.py

```diff
@@ -260,7 +260,7 @@ def _convert_listlike_datetimes(
     Parameters
     ----------
     arg : list, tuple, ndarray, Series, Index
-        date to be parced
+        date to be parsed
```

Comment:
This is indeed needs to be fixed, but it's unrelated to this PR, we are trying to having PRs to be specific about one thing.

You are more than welcome to open a separate PR fixing this :)

Reply:
Hi,

thanks for the heads up. I will open a separate pull request with this typo later today. How should we proceed with this pull request?

I have another question about the tests. One of these tests failed. I have not test this file, should I fix it then anyway or open another separate pull request for the format error there?

Reply:
> I have another question about the tests. One of these tests failed. I have not test this file, should I fix it then anyway or open another separate pull request for the format error there?

@phofl The failed CI is unrelated to your changes, and got fixed, if you merge upstream master this *should* be resolved :)

Reply:
note a minor typo *can* be included, what we don't want is more than 1 thing (major / minor) in a PR

Reply:
Got it, thank you for clarifying @jreback!